demo/grid/animated-grid.html

```diff
@@ -71,11 +70,13 @@
       }
     </style>
 
-    <template is="dom-repeat" items="[[config]]">
-      <div class$="[[_computeTileClass(item.color)]]">
-        <span>[[item.value]]</span>
-      </div>
-    </template>
+    <dom-repeat items="[[config]]">
```

Comment:
No need to wrap this in a `dom-repeat` - Polymer 2.0 will still handle `<template is=` templates parsed by polymer for backwards compatibility 👌 
This currently causes an error on `demo/grid/index.html`